src/Shell/AclShell.php

```diff
@@ -367,7 +367,10 @@ public function view()
      */
     public function initdb()
     {
-        return $this->dispatchShell('schema create DbAcl');
+        $_SERVER['argv'][1]='migrations';
+        $_SERVER['argv'][2]='migrate';
+        $_SERVER['argv'][3]='--plugin=acl';
```

Comment:
It sure is... Do we need this initdb method? Can't we just document that you need to run the migrations shell for this plugin?